cloud-haven-react-app/src/components/Notifications/NotificationBar.css

```diff
@@ -0,0 +1,70 @@
+.NotifBar {
+    width: 28%;
+    height: 100%;
+    background-color: #EDEDED;
+    margin: 2% 0 0 2%;
+    border-radius: 10px 10px 0 0;
+    padding: 0;
+}
+
+.NotifHeader {
+    height: 2.4em;
+    text-align: center;
+    background-color: #C4C4C4;
+    border-radius: 10px 10px 0 0;
+    font-size: 25px;
+    font-weight: 500;
+    padding-top: 0.3em;
+    padding-bottom: 0.3em;
+    margin-bottom: 0.65em;
+}
+
+.NotifCount {
+    background-color: #C33C54;
+    height: 1.7em;
+    width: 1.7em;
+    border-radius: 50%;
+    font-size: 21px;
+    font-weight: 500;
+    text-align: center;
+    color: #FFFFFF;
+    padding-top: 0.1em;
+    padding-bottom: 0.1em;
+    margin-left: 2%;
+    margin-right: 0;
+    position: relative;
+    top: 0.2em;
+    left: 0.2em;
+}
+
+.ShouldntNeedThisClass {
```

Comment:
This is called `ShouldntNeedThisClass`, but it's here. I assume that means it is needed? Could we name it something to help indicate what it does/what it's for?